src/test/java/com/exasol/dbbuilder/dialects/AbstractSchemaTest.java

```diff
@@ -10,6 +10,7 @@
 import org.junit.jupiter.api.Test;
 
 @SuppressWarnings("java:S5786") // this class should be public as implementation classes are in different packages
+// [utest->dsn~creating-schemas~1]
```

Comment:
I am struggling whether unit test requirement coverage in an abstract class is okay or not. On the one hand, create schema is the same for all dialects, on the other hand this way you don't annotate the individual dialect.
I think in this case it is really the best compromise since the actual test method does not appear in the derived class.